_posts/2016-06-21-foreman-community-newsletter-june-2016.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title: Foreman Community Newsletter - June 2016
+date: 2016-06-21
+author: Greg Sutcliffe
+tags:
+- foreman
+- newsletter
+excerpt: |
+  Foreman 1.12 RC2 (with Puppet 4 support!), Katello 3.0, more plugins, Red Hat
+  Summit, birthday celebrations... so much to catch up on!
+---
+
+Welcome back to the Foreman Newsletter - we hope you've had a good month! It's
+certainly been busy in the Foreman community, so let's get started...
+
+### 1.12 RC1 and RC2 released
+
+The next stable release of Foreman (1.12) has entered release candidate stage,
+and we're already up to RC2! As ever, we greatly value your testing. This
+release comes with the long-awaited Puppet 4 support, and packaging for the
+latest Ubuntu 16.04 (Xenial) OS. These are both new elements, and your feedback
+and bug reports (if any!) are welcome.
+
+As ever, you can check out the details on the [mailing list post][mlist] and
+in the [hedline features][notes]. Thanks to all who contributed bug reports and
+patches!
+
+[mlist]: https://groups.google.com/d/topic/foreman-users/NpLJGUqHMJc/discussion
+[notes]: https://theforeman.org/manuals/1.12/#Headlinefeatures
+
+### 1.11.3 bugfix released
+
+The next release in the current stable release series (1.11.3) is out now, and
+contains a fix for [CVE-2016-4451](/security.html#2016-4451), as well as many
+other fixes. See [the announcement][mlist2] for more details.
+
+[mlist2]: https://groups.google.com/d/topic/foreman-users/3tR68n5DShw/discussion
+
+### Foreman's 7th Birthday - celebrations planned!
+
+On July 13th, Foreman will be 7 years old! Since this coincides rather nicely
+with the 1.12 release (and the Puppet 4 support) I'm currently organising a
+series of events around our various geographical centres to mark the occasion.
+So far, I have events planned (but not 100% confirmed) for East-cost US, UK,
+Germany, and Israel. There'll be meetups, install days, talks and swag to be
+had! Watch [Twitter](https://twitter.com/ForemanProject) or the
+[events page](/events) to know whats confirmed!
+
+If you want to be involved in organising this effort, please contact
+[me](mailto:community@emeraldreverie.org) and I'll work something out with you
+- thanks!
+
+### Katello 3.0 (Saison) and 3.0.1 bugfix release
```

Comment:
You have written:

1.11.3 bugfix released

and

1.12 RC1 and RC2 released

So maybe you can use

Katello 3.0 (Saison) and 3.0.1 bugfix released